packages/anasazi/src/AnasaziTraceMinRitzOp.hpp

```diff
@@ -89,7 +105,7 @@ template <class Scalar, class MV, class OP>
 class TraceMinProjOp
 {
   typedef Anasazi::MultiVecTraits<Scalar,MV>    MVT;
-  const Scalar ONE; 
```

Comment:
There were some changes to code lines.  It looks like your text editor removed white space automatically.  I don't mind the changes, but it would be nice if you did Anasazi tests just to make sure everything builds OK.